arborist/server.go

```diff
@@ -972,7 +974,22 @@ func (server *Server) handleUserGrantPolicy(w http.ResponseWriter, r *http.Reque
 		_ = response.write(w, r)
 		return
 	}
-	errResponse := grantUserPolicy(server.db, username, requestPolicy.PolicyName)
+	var expiresAt *time.Time
+	if requestPolicy.ExpiresAt != "" {
+		exp, err := time.Parse(time.RFC3339, requestPolicy.ExpiresAt)
+		if err != nil {
```

Comment:
is this double parse necessary? Can you just attempt the parse to `time.RFC3339Nano`?

Reply:
honestly nano should not be necessary, I'll take that out